standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1861,6 +1861,19 @@ struct ScheduledQueryProgressInfo{
   4: optional string errorMessage,
 }
 
+struct GetFileListRequest {
+  1: optional string catName,
+  2: optional string dbName,
+  3: optional string tableName,
+  4: optional list<string> partVals,
+  6: optional string validWriteIdList
+}
+
+struct GetFileListResponse {
+  1: optional list<binary> fileListData,
+  2: optional i32 fbVersionNumber
```

Comment:
It would be good to have a comment here saying what the version number represents. Also, fbVersionNumber sounds like its a flatbuffer specific version number? Is that correct? Why does this field need to be optional?

Reply:
I agree. Here I wanted to express, that the version represents the fields in the flatbuffer file. In the first version, we'd like to store relative_path, length, last_modification_time, and file_format, but in future versions, we might want to add more fields. But you're right, this naming is confusing.